contracts/cloud-diff.yml

```diff
@@ -2,8 +2,83 @@ openapi: 3.0.0
 info:
   title: Influx Cloud API Service
   version: 2.0.1
+  description: |
+    The InfluxDB v2 API provides a programmatic interface for all interactions with InfluxDB. Access the InfluxDB API using the `/api/v2/` endpoint.
 servers:
   - url: /api/v2
+tags:
+  - name: Authentication
+    description: |
+      Use one of the following schemes to authenticate to the InfluxDB API:
+      - [Token authentication](#section/Authentication/TokenAuth)
+      - [Basic authentication](#section/Authentication/BasicAuth)
+      - [Querystring authentication](#section/Authentication/QuerystringAuth)
+      <!-- ReDoc-Inject: <security-definitions> -->
+    x-traitTag: true
+  - name: Query
+    description: |
+      Retrieve data, analyze queries, and get query suggestions.
+  - name: Write
+    description: |
+      Write time series data to buckets.
+  - name: Authorizations
+    description: |
+      Create and manage API tokens. An **authorization** associates a list of permissions to an **organization** and provides a token for API access. To assign a token to a specific user, scope the authorization to the user ID.
+x-tagGroups:
+  - name: Get started
+    tags:
+      - Authentication
+  - name: ' '
+    tags:
+      - Write
+      - Query
```

Comment:
prob makes sense to have these under get started with auth? ...the spacing between auth and query/write may look accidental or give pause